tests/test/codegen/test_machine_code_generator.cpp

```diff
@@ -248,4 +231,76 @@ TEST_CASE_METHOD(CodegenTestFixture,
       loader.loadSharedObjectObjectHash(localSharedObjFile);
     REQUIRE(hashAfter == hashBefore);
 }
+
+TEST_CASE_METHOD(CodegenTestFixture,
```

Comment:
Before, running this combination would only generate code once, and confuse the cached version of one for the other one.